packages/pwa-kit-create-app/scripts/build.js

```diff
@@ -28,7 +28,8 @@ const main = () => {
         'retail-react-app',
         'express-minimal',
         'typescript-minimal',
-        'mrt-reference-app'
+        'mrt-reference-app',
+        'base-app-extension'
```

Comment:
How hard it it to modify the code that will only make tarballs for `templates` so we don't have to have this suboptimal name.

If it's under a couple hours we should do it.